vignettes.Rmd

```diff
@@ -209,7 +209,7 @@ We discussed this syntax previously in @sec-man-key-md-features, in the context
 
 [^vignettes-4]: And, for anyone else, executing this code in the R console will open the vignette, if the host package is installed.
 
-Automatic links are generated for functions in the host package, namespace-qualified functions in another package, vignettes, and more.
+Automatic links are generated for vignettes, functions in the host package, namespace-qualified functions in another package, and more.
```

Comment:
I actually want to mention functions first, because they are the most important.

Reply:
I thought that might be the case. Something about the sentence tripped me up when I read it, but this might not be the best way to solve that.

Reply:
If you revert this change, we can merge this and just lock in the other improvements.

Reply:
Done! Sorry for the delay, I had to run to our first {usethis} pkgdocs "book club" meeting!